packages/html-webpack-plugin/index.test.js

```diff
@@ -0,0 +1,88 @@
+const Plugin = require('./index');
+const AppLoader = require('@talend/react-components/lib/AppLoader/constant').default;
+
+const DATA = {
+	head: [
+		{
+			tagName: 'link',
+			closeTag: true,
+			attributes: { rel: 'stylesheet', type: 'text/css', href: 'app.css' },
+		},
+	],
+	body: [
+		{ tagName: 'script', closeTag: true, attributes: { type: 'text/javascript', href: 'app.js' } },
+	],
+};
+
+const OPTIONS = {
+	bodyBefore: [],
+};
+
+describe('@talend/html-webpack-plugin', () => {
+	let plugin;
+	let compiler;
+	let compilation;
+	let fnc;
+	let callback;
+
+	function refresh(options) {
+		plugin = new Plugin(options);
+		compiler = {
+			plugin: jest.fn(),
+		};
+		plugin.apply(compiler);
+		compilation = compiler.plugin.mock.calls[0][1];
+		compilation(compiler);
+		fnc = compiler.plugin.mock.calls[1][1];
+		callback = jest.fn();
+	}
+
+	beforeEach(() => {
+		refresh();
+	});
+
+	it('should support options', () => {
+		refresh({ foo: 'bar' });
+		expect(plugin.options.foo).toBe('bar');
+	});
+	it('should apply call compiler/plugin function', () => {
+		expect(compiler.plugin).toHaveBeenCalled();
+		expect(compiler.plugin.mock.calls[0][0]).toBe('compilation');
+		expect(compiler.plugin.mock.calls[1][0]).toBe('html-webpack-plugin-alter-asset-tags');
+		expect(typeof fnc).toBe('function');
+	});
+	it('should call with DATA without options do not modify them', () => {
```

Comment:
This sentence is not correct --> `should not modify data when no option is provided` ?